frontend/src/components/eam-configurate-group/components/permissions-table/helpers/get-columns.helper.tsx

```diff
@@ -3,19 +3,12 @@ import {
   PermissionsTableHeader,
 } from 'common/enums/enums';
 import { Column } from 'react-table';
-import { SelectRowCell } from '../../../helpers/helpers';
 
-const getColumns = (
-  handleAddId: (id: string) => void,
-  handleRemoveId: (id: string) => void,
-  handleIsCheckedId: (id: string) => boolean,
-): Column[] => {
+const getColumns = (): Column[] => {
   return [
     {
-      Header: '',
-      accessor: 'check',
-      Cell: ({ row }): JSX.Element =>
-        SelectRowCell(row, handleAddId, handleRemoveId, handleIsCheckedId),
+      Header: PermissionsTableHeader.ACTION,
```

Comment:
actions cell should be the last one?

Reply:
![image](https://user-images.githubusercontent.com/73034554/159872521-ddecf2d4-0ff4-49ee-aeed-20f6337ff033.png)

Reply:
this action should be without name as before

Reply:
![image](https://user-images.githubusercontent.com/73034554/159936923-8b2317fa-198d-45b0-9d10-b015164bb4fe.png)